locale/en_US/mobile-appmaker.json

```diff
@@ -111,6 +119,18 @@
     "message": "From Scratch",
     "description": "Title for a blank app template. 'From Scratch' is an idiom that describes making something from the beginning with basic ingredients."
   },
+  "No data has been stored yet!": {
+    "message": "No data has been stored yet!",
+    "description": "No data has been stored yet!"
+  },
+  "noDataInfoSub": {
+    "message": "If you'd like to let users share data with you, try adding a Counter and a Submit button to your app.",
+    "description": "If you'd like to let users share data with you, try adding a Counter and a Submit button to your app."
+  },
+  "Publishing...": {
+    "message": "Publishing...",
+    "description": "Publishing... status"
```

Comment:
Can you update this to say "Shows up below header when the app is being published/ a publish is in progress